app/views/shared/_received-booking-card.html.erb

```diff
@@ -0,0 +1,20 @@
+<% @my_listings.each do |listing| %>
+  <% if listing.bookings.nil? %>
+    <p>Wow, such empty! 👶</p>
+  <% else %>
+  <% listing.bookings.each do |booking| %>
+    <div class="booking-list-card">
+      <%= link_to listing_path(listing) do %>
```

Comment:
This link probably need to go to a individual booking detail page or show a popup. Let me know your thoughts.

Reply:
I think going to an individual booking detail page is better than a popup, especially for mobile browsers.

Reply:
@worksofgenius @attinore if we are making the flow so that the user can check all their bookings on their profile page (kind of like a dashboard), I think we probably won't need an individual booking detail page?